Source/ACE.Server/WorldObjects/Player.cs

```diff
@@ -147,8 +147,10 @@ private void SetEphemeralValues()
                     IsAdmin = true;
                 if (Session.AccessLevel == AccessLevel.Developer)
                     IsArch = true;
-                if (Session.AccessLevel == AccessLevel.Envoy || Session.AccessLevel == AccessLevel.Sentinel)
+                if (Session.AccessLevel == AccessLevel.Sentinel)
                     IsSentinel = true;
+                if (Session.AccessLevel == AccessLevel.Envoy)
+                    IsEnvoy = true;
```

Comment:
```suggestion
                if (Session.AccessLevel == AccessLevel.Envoy)
                {
                    IsEnvoy = true;
                    IsSentinel = true;
                }
```
IsSentinel is important to the client, so an Envoy still needs this to be true. IsEnvoy isn't a *real* thing so sending it to the client would have no effect.